src/containers/Delete/index.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+
+import {Col, Button} from 'react-materialize';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { ActionCreators } from '@actions';
+import Report from "../../components/Report/index";
+
+class Delete extends Component {
+    constructor(props) {
+        super(props);
+
+    }
+
+
+
+    render() {
+        return (
+            <Col s={4} className='grid-example prettier'>
+                <div className="prettier">
+                    <h5>Delete comparison</h5>
+                    <hr/>
+                    <p>
+                        {
+                            this.props.reports.selected !== null ?
```

Comment:
Strict comparison can be usually simplified (if we know that null is expected in some cases) to statement like this:

...
!this.props.reports.selected ? "Raport not selected" : ... some class method or component here ...
...

In this way we actually "flip" around the whole statement, but code does the same thing. It's also just a shorthand, depends on personal code style preferences.